src/vmm/src/device_manager/legacy.rs

```diff
@@ -184,4 +174,17 @@ mod tests {
         .unwrap();
         assert!(ldm.register_devices(vm.fd()).is_ok());
     }
+
+    #[test]
+    fn test_error_messages() {
+        assert_eq!(
+            format!("{}", Error::BusError(devices::BusError::Overlap)),
+            "Failed to add legacy device to Bus: New device overlaps with an old device."
+        );
+        assert_eq!(
+            format!("{}", Error::EventFd(std::io::Error::from_raw_os_error(0)))
+                .contains("Failed to create EventFd: "),
```

Comment:
Can't assert error message exactry
Because OS error messages is different depending on the target.